From 5ff57575c98c551a6697a5dce992802e3eabcc3b Mon Sep 17 00:00:00 2001
From: Zhichao Yu <zhichao.yu@rock-chips.com>
Date: Thu, 20 Aug 2020 09:32:57 +0800
Subject: [PATCH] ARM: dts: rockchip: decrease isp reserved memory size for
 rv1126-evb-thunder-boot

We don't need so much isp reserved memory for thunder boot.
The default value is 256MB.

Signed-off-by: Zhichao Yu <zhichao.yu@rock-chips.com>
Change-Id: Ife84c37b24ae5bd882b6cbdeb612b51e7aff3db7
---
 arch/arm/boot/dts/rv1126-evb-thunder-boot.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/rv1126-evb-thunder-boot.dtsi b/arch/arm/boot/dts/rv1126-evb-thunder-boot.dtsi
index 37359ca64e93..3b5681487f72 100644
--- a/arch/arm/boot/dts/rv1126-evb-thunder-boot.dtsi
+++ b/arch/arm/boot/dts/rv1126-evb-thunder-boot.dtsi
@@ -2,6 +2,10 @@
 	clock-frequency = <1000000>;
 };
 
+&isp_reserved {
+	size = <0x08000000>;
+};
+
 &route_dsi {
 	status = "disabled";
 };
-- 
2.35.3

